tests/vmi_configuration_test.go

```diff
@@ -1541,6 +1541,44 @@ var _ = Describe("[sig-compute]Configurations", func() {
 			})
 
 		})
+
+		Context("[Serial]using defaultRuntimeClass configuration", func() {
+			It("should apply runtimeClassName to pod when set", func() {
+				By("Configuring a default runtime class")
+				runtimeClassName := "custom-runtime-class"
+				tests.CreateRuntimeClass(runtimeClassName, "custom-handler")
+				config := tests.GetCurrentKv(virtClient).Spec.Configuration.DeepCopy()
+				config.DefaultRuntimeClass = runtimeClassName
+				tests.UpdateKubeVirtConfigValueAndWait(*config)
+
+				By("Creating a new VMI")
+				var vmi = tests.NewRandomVMI()
+				vmi = tests.RunVMIAndExpectScheduling(vmi, 30)
+				Expect(err).NotTo(HaveOccurred())
+
+				By("Checking for presence of runtimeClassName")
+				pod := tests.GetPodByVirtualMachineInstance(vmi)
+				Expect(*pod.Spec.RuntimeClassName).To(BeEquivalentTo(runtimeClassName))
+
+				By("Cleaning up runtime class")
+				err = tests.DeleteRuntimeClass(runtimeClassName)
```

Comment:
Can you use a `defer` for this one or do these things in `BeforeEach` or `AfterEach`? Otherwise we would leave the runtime class in place too easy if we have a legit fail.